pkgs/data/themes/catppuccin-gtk/default.nix

```diff
@@ -37,13 +37,13 @@ assert assertUnknown;
 
 stdenvNoCC.mkDerivation rec {
   pname = "catppuccin-gtk";
-  version = "unstable-2022-02-24";
+  version = "0.2.5+date=2022-08-01";
```

Comment:
It looks like you're still using an unstable revision here -- please fix the version accordingly.

Reply:
Yes, but I thought updating the version format to the [RFC 107](https://github.com/NixOS/rfcs/pull/107) one would be better.

Reply:
I don't really think that should be done, that RFC hasn't entered its FCP yet; we still use `unstable-YYYY-MM-DD` everywhere.

@AndersonTorres Just to be sure (even though based on your last comments on the RFC, you'd agree with me), this should use the format currently in use, right?

Reply:
I understand, I'll just revert the versioning to how it was.

Reply:
I have commented a short time ago.
There are many corner cases to consider now, especially in some obscure packages.
For now I am not enforcing the format in any way.